src/main/java/io/github/mzmine/modules/io/projectload/CachedIMSRawDataFile.java

```
@@ -213,6 +213,11 @@ public boolean isContainsZeroIntensity() {
     throw new UnsupportedOperationException("Unsupported during project load.");
   }
 
+  @Override
+  public boolean isContainsEmptyScans() {
+    return false;
```

Maybe throw unsupportedOperationException?